monopod/pkg/commands/scaffold.go

```diff
@@ -91,73 +67,70 @@ func Scaffold() *cobra.Command {
 
 // runScaffold does the real work of creating the folders, files, and evaluating
 // templates.
-func (o scaffoldOptions) runScaffold() error {
+func (o scaffoldOptions) runScaffold(ctx context.Context) error {
 	if err := o.validateOptions(); err != nil {
 		return err
 	}
 
-	if err := o.createOutputFolderStructure(); err != nil {
-		return err
-	}
-
-	// this file should be written to the root of the resulting folder
-	mainTfOutfile, err := o.createOutputFile(outputMappings[MainTfTemplate])
-	if err != nil {
-		return err
-	}
-	defer mainTfOutfile.Close()
-
-	if err := o.scaffoldMainTerraform(mainTfOutfile); err != nil {
-		return err
-	}
-
-	// this file should be written in the configs directory
-	mainConfigTfOutfile, err := o.createOutputFile(filepath.Join(ConfigFolder, outputMappings[MainConfigTfTemplate]))
-	if err != nil {
-		return err
+	if err := os.MkdirAll("images/"+o.ModuleName, os.ModePerm); err != nil {
```

Comment:
I like the mkdirall, but in this case I'm not sure we should to prevent users from running it in the incorrect locations. I only say because I know of previous instances (and not only me) where some tools were not ran from the correct locations, just a thought. At least I think there's quite a few assumptions about the location of the code, and from both repos I think having the images directory there should always be the case, and not being there is a pretty good signal that there's a problem?

Reply:
That's a good point. If you run this from another directory (e.g., inside `images/apko`), it will fail with

```
Error: failed to check templates folder at path images/TEMPLATE/: stat images/TEMPLATE/: no such file or directory
```

...which isn't great, but at least it's something, and can hopefully guide people to run it in the right place.

Reply:
Perfect! As long as it fails then it's all great. Thanks for the explanation.